### DISCUSSION TITLE
Does it makes sense to have coef_ext less than 0.01?

### DISCUSSION THREAD

**Discussion by jtulak** [jmcollin78/versatile_thermostat#745] [Q&A] [answered]
Hi. I'm still calibrating the coefficients, but I can't get it to actually stabilize at the desired temperature with the coef_ext at 0.01. And it cannot be set any smaller, unless I set it to 0. It adds a degree and a half or so - probably because I have well-insulated flat with massive oldschool radiating panels placed back in time when heat insulation was not a thing, and the pipes leading heating water to other flats are going through the rooms, so I get some heating going on even when I fully close all the valves I can - about the same amount of as a sun shining through the windows, I think.

So, with external temperature coefficient set to zero it behaves reasonable, but because the temperatures so far didn't really drop bellow 0°C, I have no idea if how will the algorithm perform without coef_ext. Does it makes sense to have it enabled but at even smaller value than the current minimum 0.01 or would the effect be so small then that it makes no sense?

**Comment by jmcollin78** [comment, marked as answer]
Hello, without coef_ext the algorithm works but you will just have no compensation for outdoor cold temperatures.
This could make sense, if the room is totally isolating from outside

**Reply by jtulak** [reply]
Ok, thank you. I will keep it this way and I will see if it is enough.
